README.md

```diff
@@ -2,31 +2,372 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Explain how to parse a date from a `String` object.
+- Learn about the `DateTimeFormatter` class and formatting dates.
 
-## Introduction
+## Parsing a Date from a String
 
-A 1-2 sentence summary of what will be covered.
+In the last lesson, we saw how we could construct `LocalDate`, `LocalTime`,
+and `LocalDateTime` instances using the static methods `now()` or `of()`. But
+what if a user enters a date, and we want to take a `String` object and turn
+it into a `LocalDate`?
 
-## Topic 1
+This is where parsing comes to the rescue! We can get a `LocalDate` object by
+parsing a `String` object! The `LocalDate` class uses the default format of
+yyyy-MM-dd where the four-digit year, two-digit month, and two-digit day is
+separated by hyphens (-). The `parse()` method will take a character sequence
+argument and obtain an instance of a `LocalDate` based off of the text.
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+Let's go back to our `Person` class and modify the `setBirthday()` method to
+take in a `String` and see how the `parse()` method works:
 
-## Topic 2
+```java
+import java.time.LocalDate;
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+public class Person {
```

Comment:
The code formatting in this class seems different. Maybe add a line of spacing between the methods.